reddit_liveupdate/templates/liveupdate.xml

```diff
@@ -1,12 +1,22 @@
 <%!
-    from r2.lib.filters import spaceCompress, safemarkdown
+    from r2.lib.filters import safemarkdown
     from r2.lib.template_helpers import get_domain
+    from r2.lib.template_helpers import rss_date, rss_author
 %>
-
-<item>
-    <guid isPermaLink="true">https://${get_domain(cname=False, subreddit=False)}/live/${c.liveupdate_event._id}/updates/${thing._id}</guid>
-    <pubDate>${thing._date.strftime('%a, %d %b %Y %H:%M:%S %z')}</pubDate>
-    <description>
-        ${spaceCompress(safemarkdown(thing.body))}
-    </description>
-</item>
+<%namespace name="utils" file="utils.xml"/>
+<entry>
+    %if not thing.author.deleted:
```

Comment:
No, I thought so at first too. author here is a wrapper Template object
that only has some of the attributes of an actual Account object and
renames deleted without the underscore

On Saturday, 23 January 2016, 13steinj <notifications@github.com> wrote:

> In reddit_liveupdate/templates/liveupdate.xml
> <https://github.com/reddit/reddit-plugin-liveupdate/pull/125#discussion_r50625478>
> :
>
> >  %>
> > -
> > -<item>
> > -    <guid isPermaLink="true">https://${get_domain(cname=False, subreddit=False)}/live/${c.liveupdate_event._id}/updates/${thing._id}</guid>
> > -    <pubDate>${thing._date.strftime('%a, %d %b %Y %H:%M:%S %z')}</pubDate>
> > -    <description>
> > -        ${spaceCompress(safemarkdown(thing.body))}
> > -    </description>
> > -</item>
> > +<%namespace name="utils" file="utils.xml"/>
> > +<entry>
> > +    %if not thing.author.deleted:
>
> Stupid question here: Isn't it supposed to be thing.author._deleted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reddit/reddit-plugin-liveupdate/pull/125/files#r50625478>
> .
>